### PR TITLE
Bump libretro cores

### PR DESCRIPTION
**Pull request by paradadf**
Please make sure your PR is ready to be merged !

- [ ] You added the changes in CHANGELOG.md
- [x] You choose the right repository branch to make the PR
- [x] You described the PR as below

Fixes #XXXX

Changes :
- Not yet bumped cores are marked under the **Patches?** column. MK files were too different and I din't know what to do.
- libretro-armsnes can be deleted, as the repo doesn't exist anymore.
- The excel sheet has direct links to the relevant repos for easy access.

Related to (put here the others PR in other repositories)
Reference sheet for easy access to the repositories:
[GitHub Repos Libretro + Mupen64Plus.xlsx](https://github.com/recalbox/recalbox-buildroot/files/706089/GitHub.Repos.Libretro.Mupen64Plus.xlsx)

### CODE REVIEW

**Comment by digitalLumberjack** [comment]
Can one of the admins verify this patch?

File: package/libretro-snes9x-next/libretro-snes9x-next.mk [hunk 1]
```diff
@@ -3,15 +3,15 @@
...
 #
 ################################################################################
-LIBRETRO_SNES9X_NEXT_VERSION = dffda6f1e44522bd1e1465a49c42e407c0c3db5d
-LIBRETRO_SNES9X_NEXT_SITE = $(call github,libretro,snes9x-next,$(LIBRETRO_SNES9X_NEXT_VERSION))
+LIBRETRO_SNES9X_NEXT_VERSION = 66ba1235df30d00e018866875bc6fa77643d5663
+LIBRETRO_SNES9X_NEXT_SITE = $(call github,libretro,snes9x2010,$(LIBRETRO_SNES9X_NEXT_VERSION))
 
 define LIBRETRO_SNES9X_NEXT_BUILD_CMDS
-	CFLAGS="$(TARGET_CFLAGS)" CXXFLAGS="$(TARGET_CXXFLAGS)" $(MAKE) CXX="$(TARGET_CXX)" CC="$(TARGET_CC)" -C $(@D)/ -f Makefile.libretro platform="$(LIBRETRO_PLATFORM)"
+	CFLAGS="$(TARGET_CFLAGS)" CXXFLAGS="$(TARGET_CXXFLAGS)" $(MAKE) CXX="$(TARGET_CXX)" CC="$(TARGET_CC)" -C $(@D)/ -f Makefile.libretro platform="$(LIBRETRO_BOARD)"
```
Comment (digitalLumberjack) [inline package/libretro-snes9x-next/libretro-snes9x-next.mk:10]: Cannot work, rpi1 is not in the list of the Makefile

**Review by digitalLumberjack — CHANGES_REQUESTED** [review]

File: package/libretro-snes9x-next/libretro-snes9x-next.mk (hunk 1 above)
Comment (paradadf) [reply package/libretro-snes9x-next/libretro-snes9x-next.mk:10]: Please, feel free to edit everything! I don't have a clue what to do.

**Comment by rockaddicted** [comment]
Can be closed, let's continue in https://github.com/recalbox/recalbox-buildroot/pull/644

**Pull request closed without merging**